src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -1348,15 +1579,17 @@ public Task InvokeAsync(object[] parameters)
             }
         }
 
-        //TODO: Refactor all transient state about the connection into the ConnectionState class.
+        // TODO: Refactor all transient state about the connection into the ConnectionState class.
         private class ConnectionState : IInvocationBinder
         {
-            private volatile bool _stopping;
             private readonly HubConnection _hubConnection;
 
-            private TaskCompletionSource<object> _stopTcs;
             private readonly object _lock = new object();
             private readonly Dictionary<string, InvocationRequest> _pendingCalls = new Dictionary<string, InvocationRequest>(StringComparer.Ordinal);
+            private TaskCompletionSource<object> _stopTcs;
```

Comment:
nit: The spacing on all these members is weird

Reply:
I was trying to make the spacing more sane by keeping the readonly fields together and keeping the fields protected by the lock near the lock while also keeping the shutdown-related state fields near each other. I didn't want to add a comment because _stopTcs isn't completely protect by the _lock, but I do think the spacing is more logical than before.